file.go

```diff
@@ -38,7 +36,7 @@ func MustCreate(fileanme string) io.WriteCloser {
 //            return s
 //    }).(*AStruct)
 func OpenAnd(file string, fn func(r io.Reader) interface{}) interface{} {
```

Comment:
a bit confusing name. OpenAnd only allows reading. What about appending the same file?